caluma/form/tests/test_jexl.py

```diff
@@ -19,3 +19,22 @@
 def test_question_jexl_validate(expression, num_errors):
     jexl = QuestionJexl()
     assert len(list(jexl.validate(expression))) == num_errors
+
+
+@pytest.mark.parametrize(
+    "expression,result",
+    [
+        ('"a1"|answer', "A1"),
```

Comment:
That interface feels awkward. Why not unify the path with the question slug? Something like `"parent.form_b.b1"|answer`

Reply:
Good point. 
/cc @fkm @kaldras

Reply:
Yeah, I like this. Feels simpler and does the same!

Reply:
Fixed!